core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UniqueIDUserStoreManager.java

```diff
@@ -486,7 +486,8 @@ void updateUserListOfRoleWithID(String roleName, String[] deletedUserIDs, String
      * @throws UserStoreException If an error occurs while adding a group.
      */
     Group addGroup(String groupName, List<String> usersIDs, List<Permission> permissions,
-                   List<org.wso2.carbon.user.core.common.Claim> claims) throws UserStoreException;
+                   List<org.wso2.carbon.user.core.common.Claim> claims)
+            throws org.wso2.carbon.user.api.UserStoreException;
```

Comment:
What was the reason to remove user core UserStoreException and use org.wso2.carbon.user.api.UserStoreException ?